tox.ini

```diff
@@ -2,11 +2,15 @@
 envlist = py{37,36,35,34,27,py3,py}
 
 [testenv]
-deps = pytest-cov
-       py{37,36,35,27,py3,py}: pytest-timeout
-       py{34}: pytest-timeout<1.2.1
-       py{34}: pytest<3.3
-commands = py.test {posargs}
+deps =
+    py{37,36,35,27,py3,py}: pytest-cov
+    py{37,36,35,27,py3,py}: pytest-timeout
+    py{34}: pytest-timeout<1.2.1
+    py{34}: pytest<3.3
+    py{34}: pytest-cov==2.6.0
+commands =
+    python -m pip install -e .[watchmedo]
```

Comment:
We should always avoid installing something on the testing commands (mixing "failures" with "errors").

This is probably the best we can do to add that installation parameter: <https://tox.readthedocs.io/en/latest/config.html#conf-extras>
But an alternative would be: <https://tox.readthedocs.io/en/latest/config.html#conf-install_command>

Reply:
Way better, thanks!